ncs/encrypt_script.py

```diff
@@ -74,13 +84,19 @@ def __str__(self):
         return self.value
 
 
-KEY_IDS = {
+KEY_IDS_MKEK = {
```

Comment:
Putting myself a DNM label, as these keys are incorrect - the MKEK keys are used to encrypt keys in SICR

Reply:
Ok, fixed